extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/HibernateValidatorProcessor.java

```diff
@@ -345,11 +345,28 @@ public void build(HibernateValidatorRecorder recorder, RecorderContext recorderC
 
     @BuildStep
     NativeImageConfigBuildItem nativeImageConfig() {
-        return NativeImageConfigBuildItem.builder()
-                .addResourceBundle(AbstractMessageInterpolator.DEFAULT_VALIDATION_MESSAGES)
-                .addResourceBundle(AbstractMessageInterpolator.USER_VALIDATION_MESSAGES)
-                .addResourceBundle(AbstractMessageInterpolator.CONTRIBUTOR_VALIDATION_MESSAGES)
-                .build();
+        List<String> potentialHibernateValidatorResourceBundles = List.of(
+                AbstractMessageInterpolator.DEFAULT_VALIDATION_MESSAGES,
+                AbstractMessageInterpolator.USER_VALIDATION_MESSAGES,
+                AbstractMessageInterpolator.CONTRIBUTOR_VALIDATION_MESSAGES);
+        List<String> userDefinedHibernateValidatorResourceBundles = new ArrayList<>();
```

Comment:
Minor nitpick: you can size the list with `potentialHibernateValidatorResourceBundles.length`